acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
I tried running this locally with an EL-7 based job configuration.  The job failed at this step with the following error:

````
  dzxqhy95cr51jg5.delivery.puppetlabs.net 13:14:24$ rpm --replacepkgs -Uvh http://yum.puppetlabs.com/puppetlabs-release-puppet-el-7.noarch.rpm
    curl: (22) The requested URL returned error: 404 Not Found
    error: skipping http://yum.puppetlabs.com/puppetlabs-release-puppet-el-7.noarch.rpm - transfer failed
    Retrieving http://yum.puppetlabs.com/puppetlabs-release-puppet-el-7.noarch.rpm
````

It doesn't appear that files named like "puppetlabs-release-puppet-*" are present under http://yum.puppetlabs.com yet.  Have these repo configs not been created yet?